chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,44 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_file_v1_to_v2
+
+logger = logging.getLogger(__name__)
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        parser.add_argument(
+            "input_path",
+            help="Path to a v1 model .pt checkpoint file.",
+        )
+        parser.add_argument(
+            "output_path",
+            nargs="?",
+            help="Path at which the converted model will be saved. If not specified, it defaults to the name of the input file suffixed with '_v2.ckpt' in your current working directory.",
```

Comment:
We changed the default:
"If not specified, it defaults to the name of the input file suffixed with '_v2.ckpt' in the same parent directory."

I think how I worded this earlier could be confusing. Maybe a simple example is more clear:
"If not specified and the input file is `/path/to/checkpoint/checkpoint.pt`, the output will default to `/path/to/checkpoint/checkpoint_v2.ckpt`. 

Please choose what sounds clearer to you.